execution-engine/src/fs.rs

```diff
@@ -36,6 +38,7 @@ use std::{
 };
 // TODO: wait for icecap to support direct conversion between bytes and os_str, bypassing
 // potential utf-8 encoding check
+use log::error;
```

Comment:
Did `rustfmt` suggest moving `use log::error;` to here? If not, perhaps leave it where it was. At this location it seems to separate a comment from the code that the comment refers to! If we're keeping these `use` declarations in alphabetical order then this is the wrong place anyway.

Reply:
Hmm looks like `rustfmt` did that but when I reorder the line alphabetically and run `rustfmt` again it doesn't reorder it